tests/cucumber/features/step_definitions/datasets_steps.coffee

```diff
@@ -51,3 +51,27 @@ do ->
         .waitForExist('.remove')
         .click('.remove', assert.ifError)
         .call(callback)
+
+    @When 'the current user has a dataset called "$name"', (name)->
+      @browser
+        .execute((->
+          Meteor.userId()
+        ), ((err, ret) =>
+          assert.ifError(err)
+          @server.call('addDataset', {
+            name: name,
+            createdById: ret.value
+          })
+        ))
+
+    @When 'I go to the datasets page', ->
+      @browser
+        .waitForExist('.my-datasets-link')
+        .click('.my-datasets-link')
+        .waitForVisible(".my-datasets-header")
+
+    @When '"$name" should be listed under my datasets', (name)->
+      @browser
+        .pause(10000)
```

Comment:
I think I added this (and the Meteor.wrapAsync call) in an attempt to prevent intermittent timeouts. I can't seem to reproduce the timeouts now, so I'll remove the code.